src/main/java/capstone/mfslbackend/model/Game.java

```diff
@@ -0,0 +1,52 @@
+package capstone.mfslbackend.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@Getter
+@Setter
+//@RequiredArgsConstructor
```

Comment:
this should be replaced with @AllArgsConstructor and @NoArgsConstructor, so that way you don't need to create a constructor yourself